pkg/operator/encryptionprovider/encyrptionprovider.go

```diff
@@ -74,7 +75,7 @@ func (p *encryptionProvider) isOAuthEncryptionConfigManagedByThisOperator() (boo
 		// - we suppress the error so that the encryption controllers:
 		//    1. don't report Degraded when encryption is off
 		//    2. don't requeue when encryption is off
-		return false, nil // case 1 - OAS-O in charge
```

Comment:
see first line of the comment does not match. Probably the whole block of comment needs an update.